security/impersonating_user.rst

```diff
@@ -66,6 +66,11 @@ firewall listener:
             ),
         ));
 
+.. tip::
+
+    For using the ``switch_user` listener in a ``stateless`` firewall, set the
```

Comment:
Minor thing: in `switch_user` there's a missing closing backtick,

Reply:
fixed thanks!